src/ResourceManager/DataLakeStore/Commands.DataLakeStore/help/Add-AzureRmDataLakeStoreVirtualNetworkRule.md

```diff
@@ -0,0 +1,138 @@
+---
+external help file: Microsoft.Azure.Commands.DataLakeStore.dll-Help.xml
+Module Name: AzureRM.DataLakeStore
+online version: https://docs.microsoft.com/en-us/powershell/module/azurerm.datalakestore/add-azurermdatalakestorevirtualnetworkrule
+schema: 2.0.0
+---
+
+# Add-AzureRmDataLakeStoreVirtualNetworkRule
+
+## SYNOPSIS
+Adds a virtual network rule to the specified Data Lake Store account.
+
+## SYNTAX
+
+```
+Add-AzureRmDataLakeStoreVirtualNetworkRule [-Account] <String> [-Name] <String> [-SubnetId] <String>
+ [-DefaultProfile <IAzureContextContainer>] [-WhatIf] [-Confirm] [<CommonParameters>]
+```
+
+## DESCRIPTION
+The **Add-AzureRmDataLakeStoreVirtualNetworkRule** cmdlet adds a virtual network rule to the specified Data Lake Store account.
+
+## EXAMPLES
+
+### Example 1
+```powershell
+PS C:\> Add-AzureRmDataLakeStoreVirtualNetworkRule -Account "dls" -VirtualNetworkRuleName "myVNET" -VirtualNetworkSubnetId "subnetId"
```

Comment:
@lewu-msft please make sure that the examples (in the files you are adding) contain the corresponding output from the commands that are being run. This gives users a better idea of the properties that the returned object has

Reply:
@lewu-msft ping on this comment